demo/events.js

```diff
@@ -19,6 +19,13 @@ function addEvents(grid, id) {
     console.log(g + 'dragstart ' + el.textContent + ' pos: (' + node.x + ',' + node.y + ') vs (' + x + ',' + y + ')');
   });
 
+  grid.on('drag', function(event, el) {
+    let node = el.gridstackNode;
+    let x = el.getAttribute('gs-x');

```

Comment:
actually I don't know why I have el.getAttribute('gs-x') vs node.x as they should be the same....maybe for debugging. I should make that clearer...